### PR TITLE
Change packet beat to metricbeat

### PR DESCRIPTION
**Pull request by liubin**
I think this is a typo.

### CODE REVIEW

File: libbeat/docs/shared-command-line.asciidoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ Overwrite an individual setting in the config file. For example:
 +
 ["source","sh"]
 ----------------------------------------------------------------------
-sudo ./packetbeat -c packetbeat.yml -E name=mybeat
```
Comment (ruflin) [inline libbeat/docs/shared-command-line.asciidoc]: This is used by all beats, not only packetbeat or metricbeat. 

@dedemorton Can we use a variable "inside" the code part?

Comment (dedemorton) [inline libbeat/docs/shared-command-line.asciidoc]: Yeppers! This should work:

`````
["source","sh",subs="attributes"]
 ----------------------------------------------------------------------
-sudo ./{beatname_lc} -c {beatname_lc}.yml -E name=mybeat
----------------------------------------------------------------------
`````

Comment (ruflin) [inline libbeat/docs/shared-command-line.asciidoc]: @liubin Do you want to update your PR with the proposal above or should we take over? Thanks a lot for bringing this up.

Comment (liubin) [inline libbeat/docs/shared-command-line.asciidoc]: @ruflin I'll close this PR.

**Comment by dedemorton** [comment]
@liubin Thanks very much for bringing this to our attention. I've created PR #2934 to resolve this issue.

**Pull request closed without merging**